_posts/2020-02-14-running-a-great-socratic-seminar.md

```diff
@@ -0,0 +1,217 @@
+- Announce your event in the usual planes online (Reddit, Twitter, etc.).
+- Connect with organizers of Socratic Seminars in other cities and tap into
+  their networks.
+- If you're aware of specific people within a reasonable distance of your
+  meetup who would be strong candidates for participating in the discussion,
+  invite them individually! Good candidates are domain experts or people with
+  very specific interests in an area that you would like to cover.
+- If there are interesting conferences, hackathons, etc. in your city, organize
+  an event the same week and invite any visiting experts.
+
+If you specifically invite people with expert knowledge, let them know that
+you'd like their participation in the discussion -- people will not naturally
+come prepared to engage, so you have to foster that environment.
+
+### Discovering Content
+
+The next hurdle is content discovery and collation. Topics we often look at
+include:
+
+- Critical vulnerabilities in popular hardware and software
+- Network statistics and phenomena (e.g. oscillations in the
+  mempool/difficulty/UTXO set, reorgs, 51% attacks, double spends)
+- Mailing list posts with interesting proposals or critiques
+- Pull requests from popular repositories that include substantial changes
+- Research papers relevant to bitcoiners
+- Blog posts on privacy, software development, cryptography and more
+
+There are many Socratic style meetups worldwide which post their content
+online; going through their recent events will give you a lot of materials to
+choose from (e.g. [New York](https://bitdevs.org/events.html), [San
+Francisco](https://www.sfbitcoindevs.org), [Berlin](https://bitdevs.berlin)).
+The [Optech newsletter](https://bitcoinops.org/en/newsletters/) is also a good
+source of relevant content.
+
+Tailor the content to what the attendees will find interesting. It's OK if you
+aren't having the exact same discussion that another city is having. Maybe your
+group is more excited about privacy because of the part of the world you are
+in, or maybe you are worried about regulation because you live in a financial
+capital. Make it your own and tailor it to the people that show up
+consistently.
+
+### Preparation
+
+With content selected, it's time to prepare yourself to lead the discussion.
+First, read and highlight the material. There are highlighting plugins for all
+browsers. Use highlights to emphasize important content on the page -- they're
+a good mnemonic device for the host and they help drive discussion. That said,
+be careful not to use the highlights as a crutch -- if you exclusively read
+word-for-word from the links and do not engage the crowd, you'll lose their
+attention and discussion will fall flat.
+
+Generally, preparing slides isn't useful or necessary -- it takes a long time
+to prepare, and people want to see the source material anyway. 
+
+Arrange the tabs in your browser from left to right, starting with the things
```

Comment:
Could probably drop "from left to right" to leave it up to the reader to choose whatever ordering they find most natural (e.g. a Hebrew reader might prefer right to left).